src/components/html-injection/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const widgetName = "html-injection";
+export const widgetDisplayName = "HTML injection";
```

Comment:
I would change name "HTML injection" which doesn't look friendly for end-users. Maybe something like "Custom HTML block"?